otter/test/test_metrics.py

```diff
@@ -263,8 +263,8 @@ def test_get_all_metrics(self):
         """
         Metrics are returned based on the requests done to get server info.
         """
-        # Maybe this could use a parameterized "get_scaling_group_servers" call
-        # to avoid needing to stub the nova responses, but it seems okay.
+        # Maybe this could use a parameterized "get_all_scaling_group_servers"
+        # call to avoid needing to stub the nova responses, but it seems okay.
         servers_t1 = {
             'g1': ([_server('g1', 'ACTIVE')] * 3
                    + [_server('g1', 'BUILD')] * 2),
```

Comment:
Lint seems to be unhappy about this `+` and wants it to go on the previous line.